java/ql/lib/semmle/code/java/security/XmlParsers.qll

```diff
@@ -991,28 +991,6 @@ class SafeTransformer extends MethodAccess {
   }
 }
 
-/*
- * SAXTransformer: https://cheatsheetseries.owasp.org/cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.html#saxtransformerfactory
```

Comment:
Undo this change: this file should be unchanged except exposing `SafeTransformerFactoryFlowConfig` which is acceptable, then the experimental query should add the extra sink.